src/payment/strategies/sdk/handler/handler.ts

```diff
@@ -0,0 +1,52 @@
+
+import { handleContinueResponse, isContinueResponse } from './continue';
+import { handleErrorResponse } from './error';
+import { handleFailedResponse, isFailedResponse } from './failed';
+import { handleSuccessResponse, isSuccessResponse } from './success';
+
+/*
+    This will likely need some local state
+    for storing payment method data only available at time of instantiation
+    so will probably need to become a class
+*/
+
+export const handler = async (response: Response): Promise<void> => {
```

Comment:
I'd probably make those functions injectable so they can unit tested without having to use `jest.mock` to hijack `import/require`.